powerstore/resource_host_group.go

```diff
@@ -138,6 +138,15 @@ func (r *resourceHostGroup) Create(ctx context.Context, req resource.CreateReque
 		return
 	}
 
+	valid, errmsg := r.fetchByName(&plan)
+	if !valid {
```

Comment:
Can we check error condition on errmsg instead of valid, since we encountered similar issue with checkmarx earlier as well and we had to fix the same.

Reply:
done